### DISCUSSION TITLE
Export to PNG and SVG

### DISCUSSION THREAD

**Discussion by paulomorgado** [structurizr/cli#145] [Q&A]
I can download diagrams as PNG and SVG from Structurzr Lite.

It would be nice to be able to do the same with the CLI.

**Comment by simonbrowndotje** [comment]
The CLI is a Java application and Structurizr Lite's diagrams are generated via a web browser ... so this isn't possible. You can use Node + Puppeteer though -> https://github.com/structurizr/puppeteer
